contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -1006,7 +1019,21 @@ var MultiLanguageUploadSlot = FormatSlot.extend({
         }
       )
     );
-    _.defer(this.create_dropzone);
+
+    if (this.language) {
+      var settings = {
+        headers: {
+          Language: this.language,
+        },
+      };
+
+      if (this.model.get('id') === 'video_subtitle') {
+        settings.url = window.Urls.subtitle_upload();
```

Comment:
Ah interesting- would it be possible to use a more flexible way to handle uploading files for multilanguage format presets? Ideally, we would be able to just add to le_utils and have Studio be able to handle any new presets automatically, so it might be worth making this a `multilanguage_file_upload` endpoint or building on the file_upload endpoint (if it doesn't add too much to the endpoint)

Reply:
Created the `multilanguage_file_upload`